docs/apps/interchain-accounts/client.md

```diff
@@ -0,0 +1,232 @@
+<!--
+order: 7
+-->
+
+# Client
```

Comment:
I added this file for all CLI and grpc queries following the example of the SDK (e.g. for [`x/bank`](https://github.com/cosmos/cosmos-sdk/tree/main/x/bank#client)).